mne/externals/FieldTrip.py

```diff
@@ -309,7 +309,7 @@ def getHeader(self):
                 (chunk_type, chunk_len) = struct.unpack(
                     'II', payload[offset:offset + 8])
                 offset += 8
-                if offset + chunk_len < bufsize:
+                if offset + chunk_len > bufsize:

```

Comment:
btw, my neuromag2ft (from fieldtrip repo) does not accept --file. You must have some bonus version of it? :)